source/jquery.flot.time.js

```diff
@@ -141,6 +218,8 @@ API.txt for details.
 
         if (opts && opts.timeBase === 'seconds') {
             ts *= 1000;
+        } else if (opts.timeBase === 'microseconds') {
```

Comment:
We should provide documentation for this option. Probably, [here](https://github.com/flot/flot/blob/master/API.md#time-series-data), and [here](https://github.com/flot/flot/blob/master/source/jquery.flot.time.js#L19). I don't think we did a good job of bubbling this option up to the docs in the first link, and now would be a good time to do so.

Reply:
I updated the API.md and the comment in the latest commits I pushed today.